tests/conftest.py

```diff
@@ -1,3 +1,7 @@
+import webbrowser
+from http.server import HTTPServer
+from unittest.mock import Mock
```

Comment:
I think this is no longer used

Reply:
Everything except `from http.server import HTTPServer` has been removed.